### ISSUE TITLE
'Cycle display mode' shortcut is not reassignable

### ISSUE THREAD

**Issue by ejiektpobehuk** [elementary/settings-keyboard#262]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Describe the bug
It's not currently possible to change **Cycle display mode** shortcut to a custom one.
When custom action is set to `⌘+P`  **Cycle display mode** action is still being called.

It's very visible on dualscreen setup with non-default configuration (I have one screen rotated).

Came from [reddit post]( https://www.reddit.com/r/elementaryos/comments/e9qazx/cycle_display_mode_shortcut/)

### To Reproduce
1. Go to `System Settings->Keyboard->Shortcuts`
2. Set custom shortcut to `⌘+P` (`io.elementary.terminal` for example)
3. Press `⌘+P`

### Expected behavior
Terminal window opens

## Platform Information

 - elementary OS 5.1
 - Hera

- [x] I'm using the latest released stable version

## Additional context

Works just fine if you set **Cycle workspace** to `⌘+P`

In **Shortcut Overview** top right position is occupied with `⌘+P` shortcut:

* **Cycle display mode** when nothing is set
*  **Cycle workspace** when  **Cycle workspace** is set to it
*  **Cycle display mode** when custom command is set to it

PS. I think it's a good idea to make **Cycle display mode** visiable and reassignable in  `System Settings->Keyboard->Shortcuts->System'

**Comment by danrabbit** [comment]
The shortcut Super + P is not reassignable, so we should disallow/warn here

**Comment by ejiek** [comment]
I've done a bit more testing =]

| Option            | Action                      |
| ---               | ---                         |
| Switch to new     | **Cdm**s (!)                |
| Cycle workspaces  | cycles workspaces           |
| Multitasking View | shows **multitasking view** |
| Switch left       | switches left               |
| Switch right      | switches right              |
| Switch to first   | switches to first           |
| Custom action     | **Cdm**s (!)                |

**Comment by cassidyjames** [comment]
Yeah, it's something like that Microsoft wrote it into the "Designed for Windows" or whatever spec that Super+P had to cycle display modes. So instead of implementing it in software, a bunch of OEMs implemented it in keyboard firmware so that Super+P actually sends a different keycode and thus is not reliably reassignable.

**Comment by ejiek** [comment]
Thanks for the response!

Though it's strange that for some actions remap works just fine and for other it does not =\ 

Should i try to collect key codes for my system?

**Comment by lenemter** [comment]
Closing as out of scope
